File: homework/03/10.js
/* 物件遍歷

建立一個物件 scores = { Alice: 85, Bob: 92, Charlie: 78 }。
使用 for...in 迴圈遍歷 scores 物件，輸出 "Alice 的分數是 85" 這樣的資訊。 */
let scores = {
    Alice: 85,
    Bob: 92,
    Charlie: 78
};

for (let name in scores) {
    console.log(`${name} 的分數是 ${scores[name]}`);
}

File: homework/08/10.py
def summary(data):
    for student in data:
        name = student['name']
        scores = student['scores']
        total = sum(scores)
        avg = round(total / len(scores), 1) if scores else 0
        print(f"{name} 總分: {total}, 平均: {avg}")

students = [
    {'name': 'Andy', 'scores': [90, 80, 70]},
    {'name': 'jack', 'scores': [60, 85, 95]},
    {'name': 'alan', 'scores': [160, 185, 195]},
]

summary(students)
